webapp/src/components/end_call_modal/component.scss

```diff
@@ -0,0 +1,42 @@
+%button-end-call-modal {
```

Comment:
ahhh... nooo... scss.... :( 
But it doesn't matter I guess. I have to go through and convert to styled-components someday later.

Reply:
Sorry :see_no_evil: